src/lib/helpers/queryKeys.ts

```diff
@@ -28,12 +28,18 @@ export const queryKeys = createQueryKeyStore({
   concentrators: {
     list: ({ chainId }: { chainId: number }) => [chainId],
     detail: ({
+      targetAssetId,
```

Comment:
Do we need all this added info because of the API? Suggests the API should be changed if it is that much more complicated to use vs. the on-chain data.

Reply:
I don't think he can do it differently, I mean he needs to know the target asset and the primary asset, with the chainId. We could maybe combine them and use one ID that represents the target asset/the primary Asset and the chainId. In the smart contract, we use the target asset with the primary asset too.

Reply:
Yeah, my mistake I was confused. That makes sense.

I still think that we should be using addresses instead of IDs because this information can be universal to the chain and the backend... but you're working with what you've got on the backend I understand.

Reply:
Yes, the addresses are the best choice IMO, the targetAssetId and the concentratorId here are proper to the BE. Make sense to ask the BE to use the addresses.